<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>___________</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">font-size: 20pt; font-weight: bold; color: white;</string>
      </property>
      <property name="text">
       <string>ðŸ—‚ Archive Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">
        background-color: #2b2b2b;
        color: #f0f0f0;
        border-radius: 8px;
        font-size: 11pt;
        padding: 8px;
       </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="createButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: #3b82f6;
          border: none;
          border-radius: 6px;
          color: white;
          font-size: 11pt;
          padding: 10px;
         </string>
        </property>
        <property name="text">
         <string>ðŸ›  Create Archive</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="extractAllButton">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
          background-color: #3b82f6;
          border: none;
          border-radius: 6px;
          color: white;
          font-size: 11pt;
          padding: 10px;
         </string>
          </property>
          <property name="text">
           <string>ðŸ“¦ Extract All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="openButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: #3b82f6;
          border: none;
          border-radius: 6px;
          color: white;
          font-size: 11pt;
          padding: 10px;
         </string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="extractButton">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: #3b82f6;
          border: none;
          border-radius: 6px;
          color: white;
          font-size: 11pt;
          padding: 10px;
         </string>
        </property>
        <property name="text">
         <string>ðŸ“¦ Extract File(s)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
